CHANGELOG.md

```diff
@@ -72,7 +72,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed [#1716](https://github.com/JabRef/jabref/issues/1716): `@`-Symbols stored in BibTeX fields no longer break the database
 - Fixed [#1499](https://github.com/JabRef/jabref/issues/1499): {} braces are now treated correctly in in author/editor
 - Fixed [#1531](https://github.com/JabRef/jabref/issues/1531): `\relax` can be used for abbreviation of author names
- 
```

Comment:
I thought so. Well, it is not entirely consistent (there are similar
entries in the current development section of the ChangeLog), so more for
next time.

(I'm thinking that maybe someone should traverse the ChangeLog and sort
issues at some stage. Then one can rephrase some entries, which are just
copies of the issue, I've done that, and possibly remove "doubtful" entries
like this. Hard to always know and keep track of though.)